tsdb/index/index.go

```diff
@@ -526,7 +526,7 @@ func (w *Writer) finishSymbols() error {
 	symbolTableSize := w.f.pos - w.toc.Symbols - 4
 	// The symbol table's <len> part is 4 bytes. So the total symbol table size must be less than or equal to 2^32-1
 	if symbolTableSize > math.MaxUint32 {
-		return errors.Errorf("symbol table size exceeds 4 bytes: %d", symbolTableSize)
+		return errors.Errorf("symbol table size exceeds %d bytes: %d", uint32(math.MaxUint32), symbolTableSize)
```

Comment:
```suggestion
		return errors.Errorf("symbol table size exceeds %d bytes: %d", math.MaxUint32, symbolTableSize)
```
https://go.dev/play/p/zmUN1mQzS_V

Reply:
ooh 32 bit archs ..